jump-start/solr9/cloud-config-set/solrconfig_query.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+  Document Cache
+  7.0.0
+-->
+<documentCache class="solr.LRUCache" size="512" initialSize="512" autowarmCount="0"/>
+
+<!--
+  Field Value Cache
+  7.0.0
+-->
+<fieldValueCache class="solr.FastLRUCache" size="512" autowarmCount="128" showItems="32"/>
+
+<!--
+  Filter Cache
+  7.0.0
+-->
+<filterCache class="solr.FastLRUCache" size="512" initialSize="512" autowarmCount="0"/>
+
+<!--
+  Per Segment Filter Cache
+  7.0.0
+-->
+<cache name="perSegFilter" class="solr.search.LRUCache" size="10" initialSize="0" autowarmCount="10" regenerator="solr.NoOpRegenerator"/>
+
+<!--
+  Query Result Cache
+  7.0.0
+-->
+<queryResultCache class="solr.LRUCache" size="512" initialSize="512" autowarmCount="0"/>
```

Comment:
> LRUCache, LFUCache and FastLRUCache are deprecated and will be removed in Solr 9.0. Users should switch their configurations to use CaffeineCache instead.

Following lead from https://github.com/IQSS/dataverse/issues/7662 I think removing the class would be enough to remove those deprecated cache references.